var/spack/repos/builtin/packages/intel-tbb/package.py

```diff
@@ -112,6 +112,12 @@ class IntelTbb(Package):
     # Some very old systems don't support transactional memory.
     patch("disable-tm.patch", when='~tm')
 
+    # Add support for building on arm64 macOS,
+    # also included in hombrew and already available upstream:
+    # https://github.com/oneapi-src/oneTBB/pull/258
+    # https://github.com/oneapi-src/oneTBB/commit/86f6dcdc17a8f5ef2382faaef860cfa5243984fe.patch?full_index=1
+    patch("macos-arm64.patch")
```

Comment:
Which versions does this patch apply to? I assume it doesn't apply to every version, and it won't be needed in newer versions. You can test this with `spack patch intel-tbb@X.Y`

Reply:
I checked that applies from 4.4.4 up to 2021.1-beta05. Because of my compiler being to new, I couldn't check older versions. I also build-tested all the versions that do not conflict with clang.

Reply:
I would change these ranges to `:2021.0` (drop the beta version). We likely won't add the beta versions of the package, and Spack sorts beta versions higher than releases at the moment.

Reply:
I just pushed an update with upper version limits set to 2021.0